listenbrainz/db/external_service_oauth.py

```diff
@@ -1,13 +1,17 @@
+from typing import List
+
+from data.model.external_service import ExternalServiceType
 from listenbrainz import db, utils
 import sqlalchemy
 
 
-def save_token(user_id, service, access_token, refresh_token, token_expires_ts, record_listens, scopes):
+def save_token(user_id: int, service: ExternalServiceType, access_token: str, refresh_token: str,
+               token_expires_ts: int, record_listens: bool, scopes: List[str]):
     """ Add a row to the external_service_oauth table for specified user with corresponding tokens and information.
 
     Args:
         user_id (int): the ListenBrainz row ID of the user
```

Comment:
if we have types in the function definition then they're not required in the docstrings